### PR TITLE
Nandini

### PR DESCRIPTION
**Pull request by Nandiniverma2024**

### CODE REVIEW

File: README.md
Comment (tanishdotdev) [inline README.md:1]: 1. The colors are already defined in tailwind, Only define the colors which are custom and added in tailwind config file.
2. Update this after you make changes in the files.

File: src/atoms/atoms-nandini/headings/one/index.astro
Comment (tanishdotdev) [inline src/atoms/atoms-nandini/headings/one/index.astro:1]: 1. You have not added the text size like this, headings/one/3xl/index.astro.
2. Remove padding.

File: src/atoms/atoms-nandini/paragraphs/white/index.astro
Comment (tanishdotdev) [inline src/atoms/atoms-nandini/paragraphs/white/index.astro:1]: 1. Remove margins and padding.
2. Add Text size like paragraphs/base/white/index.astro

File: src/components/home/lets-talk/index.astro
Comment (tanishdotdev) [inline src/components/home/lets-talk/index.astro:1]: 1. Remove text-center from section tag.
2. Why are you using space-x-1 in line 23?

File: src/layouts/layout/index.astro
Comment (tanishdotdev) [inline src/layouts/layout/index.astro:1]: Isn't the font already added in the layout?

It is already defined in tailwind config file.

File: tailwind.config.mjs
Comment (tanishdotdev) [inline tailwind.config.mjs:1]: Remove all these, as these are already defined in tailwind, We only add colors which are not defined in tailwind.

**Review by tanishdotdev — CHANGES_REQUESTED** [review]

File: src/atoms/atoms-nandini/headings/one/index.astro
Comment (Nandiniverma2024) [reply src/atoms/atoms-nandini/headings/one/index.astro:1]: Should I need to make another folder 3xl and then index.astro  
Like heading->one->3xl->index.astro

File: src/layouts/layout/index.astro
Comment (Nandiniverma2024) [reply src/layouts/layout/index.astro:1]: yes, actually I have seen tailwind after importing it in tailwind config (I forgot to remove at time time)

I Remove it now

**Pull request closed without merging**